packages/lesmis/src/components/DashboardExportModal/pages/styles.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const DashboardTitleContainer = styled.div`
```

Comment:
Good idea creating a new component for this. I would prefer to move it to the DashboardExportModal/components folder and rename the file to DashboardTitleContainer.js since that is more consistent with the conventions in lesmis.